src/network/http_server.rs

```diff
@@ -297,18 +297,17 @@ pub struct CastsByParentRequest {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ReactionRequest {
     fid: u64,
-    #[serde(rename = "reactionType")]
-    reaction_type: ReactionType,
-    #[serde(rename = "targetCastId", skip_serializing_if = "Option::is_none")]
+    #[serde(rename = "target_fid", skip_serializing_if = "Option::is_none")]
     target_cast_id: Option<CastId>,
-    #[serde(rename = "targetUrl", skip_serializing_if = "Option::is_none")]
+    // I think this is actually a different type not just a rename - worth digging into more
+    #[serde(rename = "target_hash", skip_serializing_if = "Option::is_none")]
     target_url: Option<String>,
```

Comment:
Yes, they are different. We should support both target_url and target_hash. Looks like hubs only support target_hash right now, so if you want to pick one, you can pick that and rename this field to `target_hash`

Reply:
I added support for both - that should still be backwards compatible and it sounds like this is more in line with desired functionality.